test/adapter.spec.js

```diff
@@ -253,6 +253,14 @@ describe('jasmine adapter', function () {
     it('should report time for every spec', function () {
       var counter = 3
 
+      function copySpecResult() {
```

Comment:
I think this can just be 
```js
return spec.result.slice(0);
```
But in any case pass the `spec` as an arg to make it clearer.